### DISCUSSION TITLE
Button size increas on Renovate PRs

### DISCUSSION THREAD

**Discussion by wyardley** [renovatebot/renovate#22719] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

The old buttons for "age" / "adoption" etc. were fairly small / normal size (like the typical button sizes you see on a readme, and proportional to the text to their left), but now are showing up supersized for me. Is this intentional?

<img width="814" alt="image" src="https://github.com/renovatebot/renovate/assets/10714426/a24a9f7d-043c-475d-ac92-759214e7d71b">

### What you tried so far.

* Checked in Chrome
* Checked to make sure browser magnification settings were normal

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
yes

**Reply by wyardley** [reply]
Ok, thanks. I'll close this out.
FWIW, to me it looks very odd, but not sure if it's the resolution or the proportion relative to the items to the left... maybe both? I think the proportions looked better how it was before.

**Reply by viceice** [reply]
but they were too small on big retina displays, so unreadable there
